src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -399,7 +420,7 @@ private static void writeTokensList(Sink sink, List<String> tokens, String token
         }
         else {
             sink.rawText(INDENT_LEVEL_18);
-            sink.text(SiteUtil.DOT);
+            sink.rawText(SiteUtil.DOT);
```

Comment:
Why was this changed? Is this is connected to https://github.com/checkstyle/checkstyle/pull/13971#discussion_r1388842218 ?

Reply:
I am surprised how it got removed when i can see my locak having those changes. i will rebase again and push.